tests/test_path_field.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from django.core.exceptions import ValidationError
+from django_ltree.fields import PathValue
+
+from taxonomy.models import Taxonomy
+
+def test_validate_path_field_root_path_valid():
```

Comment:
Will be nice to have it parametrized, all the tests are the same with just different conditions.

Maybe something like this:

```
@pytest.mark.parametrize(["path", "valid"], [
    ("00_00a", True),
    ("00$00", False),  # Setting it to false to check if the validation error 
    ("00000a.00000b", True),
    ("00000a+00000b", False),
])
def test_path_validation(path, valid):
    taxonomy = Taxonomy(name='test', path=PathValue(path))
    if valid:
        taxonomy.full_clean()
    else:
        with pytest.raises(ValidationError) as excinfo:
            taxonomy.full_clean()
```